Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM alpine:3.7
+FROM alpine:3
```

Comment:
I'd like to be as specific as possible so that builds are reproducible. Could you please help me to change this to `alpine:3.13.5` (which [seems to be the latest tag](https://hub.docker.com/_/alpine?tab=tags))?

Reply:
Even better! I updated the tag